packages/react-form-state/CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
+## [0.9.0]
+
+### Removed
```

Comment:
I think this is more of an `Update`

Reply:
You're correct, I just re-read the changelog guidelines and this is definitely `Changed`. I'll update it after merging and before releasing, so as not to lose the ✅.